persistence/jpa/src/test/java/org/infinispan/persistence/jpa/AbstractJpaStoreTest.java

```diff
@@ -59,7 +62,13 @@ protected JpaStore createCacheStore() {
       store.start();
 
       assertNotNull(store.getEntityManagerFactory());
-      assertTrue(store.getEntityManagerFactory() instanceof org.hibernate.internal.SessionFactoryImpl);
+
+      //EAP 7.1 uses Hibernate 5.1
+      if(HIBERNATE_VERSION < 5.2) {
```

Comment:
Nitpick, but can you fix the formatting on the comment and the if to have a leading space.